modules/programs/wpaperd.nix

```
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.wpaperd;
+  tomlFormat = pkgs.formats.toml { };
+in {
+  meta.maintainers = [ maintainers.Avimitin ];
+
+  options.programs.wpaperd = {
+    enable = mkEnableOption "wpaperd";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.wpaperd;
+      defaultText = literalExpression "pkgs.wpaperd";
+      description = "wpaperd package to install.";
+    };
```

Can use the `mkPackageOption` function to simplify this a bit:

```suggestion
    package = mkPackageOption pkgs "wpaperd" { };
```